src/components/common/Button.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <button
+    class="bg-main rounded-sm text-white font-normal py-1 px-10 text-base"
+    :class="bindClasses()"
+  >
+    {{ label }}
+  </button>
+</template>
+
+<script>
+export default {
+  name: "Button",
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    isBig: {
+      type: Boolean,
+      default: false,
+    },
+    isWarning: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  setup(props) {
+    const bindClasses = () => {
+      const classes = [];
```

Comment:
no ok, ale zmieściło by się to w htmlu na 
`:class = 
{ 'px-40': isBig,
'bg-warning text-gray-900': isWarning
 }
`
ma to taką zaletę, że trzymasz css razem z htmlem a nie mieszasz go z jsem, tzn wszystko jest w dedykowanym dla siebie miejscu w kodzie